contracts/test/recovery-test.ts

```diff
@@ -99,13 +111,16 @@ describe("Recovery", async function () {
     await fx.advanceTimeBy(safetyDelaySeconds / 2); // wait half the time
     await (await blsWallet.setAnyPending()).wait();
 
-    const safeKey: PublicKey = await wallet2.blsWalletSigner.getPublicKey(
+    const signedAddress = await signWalletAddress(
```

Comment:
Naming of signatures as discussed, eg `addressSignature`

Reply:
Done